apps/lfx-one/src/app/modules/profile/components/profile-edit-dialog/profile-edit-dialog.component.html

```
@@ -73,15 +73,34 @@ <h3 class="text-base font-semibold text-slate-900 mb-4" data-testid="profile-edi
+                [showClear]="true"
+                [filter]="true"
+                [loading]="loadingWorkExperiences()"
+                styleClass="w-full"
+                data-testid="profile-edit-dialog-organization">
+              </lfx-select>
+            } @else {
+              <lfx-select
+                size="small"
+                [form]="profileForm"
+                control="organization"
+                id="edit-organization"
```

HTMLHint (1.9.2)

[error] The id value [ edit-organization ] must be unique.

(id-unique)